app/assets/scripts/components/map/measurements-layer.js

```diff
@@ -23,115 +16,75 @@ export default function MeasurementsLayer({
   let match = useRouteMatch();
 
   const countryFilter = ['==', ['get', 'country'], country];
-  const circlesFilter = ['==', ['get', 'sensorType'], 'reference grade'];
-  const squaresFilter = ['==', ['get', 'sensorType'], 'low-cost sensor'];
-
-  const circlesCountryFilter = ['all', countryFilter, circlesFilter];
-  const squaresCountryFilter = ['all', countryFilter, squaresFilter];
+  const iconMatch = [
+    'match',
+    ['get', 'sensorType'],
+    'low-cost sensor',
+    'square',
+    'reference grade',
```

Comment:
```suggestion
    'reference',
```
The backend settled on `reference`.